### ISSUE TITLE
new rule `border-width-no-named-notation`

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#992]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

https://developer.mozilla.org/ru/docs/Web/CSS/border-width
Disable usage `thin`, `medium`, `thick` keywords.

> Which rule, if any, is this issue related to?

`border-width-no-named-notation`

> What CSS is needed to reproduce this issue?

```css
.foo {
  border-width: thin;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "border-width-no-named-notation": true
  }
}
```

> Which version of stylelint are you using?

 `5.2.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

no matter

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no matter

> What did you expect to happen?

Throw error if you use keywords.

> What actually happened (e.g. what warnings or errors you are getting)?

Nothing.

**Comment by jeddy3** [comment]
Wow, who knew there were keywords values for border width? I'm always learning something new about the CSS specs. That's what I love about this project :)

As always, thanks for the suggestion @evilebottnawi .

For the other `*-notation` rules we've allowed explicit options e.g. `"numeric"|"named"`. I think we should follow a similar pattern for here *if* users would ever want to enforce "named" values. Otherwise, if the intent is just to disallow named keywords for `border-width`, then can't that already be achieved using the `property-value-black` rule?

**Comment by evilebottnawi** [comment]
@jeddy3 Yes, agree rare thing. I think you are right at the moment we do not need this rule. `property-value-blacklist` best solution for disable usage this rare keywords. If anyone will ever need in this rule, it will be possible to reopen it and make it in the future.
